docs/guides/insar_product_guide.md

```diff
@@ -186,7 +186,12 @@ All of the phase differences in a wrapped interferograms lie between -&#960 and
 
 Before the interferogram can be unwrapped, it must be filtered to remove noise. This is accomplished using an adaptive spectral filtering algorithm. This adaptive interferogram filtering aims to reduce phase noise, increase the accuracy of the interferometric phase, and reduce the number of interferogram residues as an aid to phase unwrapping. In this case, residues are points in the interferogram where the sum of the phase differences between pixels around a closed path is not 0.0, which indicates a jump in phase.
 
-Another step before unwrapping is to create a coherence mask to guide the phase unwrapping process. The coherence is estimated from the normalized interferogram and the co-registered intensity images using an MLI estimator with rectangular weighting with a 5x5 moving window. This file has values from 0.0 (total decorrelation) to 1.0 (perfectly coherent).  The coherence is then turned into a mask wherein all pixels are either 0 (don't unwrap) or 1 (unwrap). Any input pixel with a coherence less than 0.1 or an intensity below 0.2 are set to zero and not used during unwrapping.
+#### Masking
+Another step before unwrapping is to create a validity mask to guide the phase unwrapping process. This mask is generated from the coherence and amplitude (backscatter intensity) values. 
+
+Coherence is estimated from the normalized interferogram and the co-registered intensity images using an MLI estimator with rectangular weighting with a 5x5 moving window. This file has values from 0.0 (total decorrelation) to 1.0 (perfectly coherent). The coherence is then turned into a mask wherein all pixels are either 0 (don't unwrap) or 1 (unwrap). Any input pixel with a coherence value less than 0.1 or an intensity value less than 0.2 is set to zero and not used during unwrapping.
+
+When the water masking option is applied, the validity mask is further amended to apply 0 values to any pixels classified as water in the water mask. In some cases, pixels over water may still meet the coherence and amplitude threshold criteria for inclusion, even though they are not valid for use during phase unwrapping. When processing scenes with extensive coverage by coastal waters or large inland waterbodies, there can be erroneous phase jumps introduced if unwrapping proceeds over water as if it were land. In such cases, choosing the option to apply the water mask can significantly improve the results. 
```

Comment:
I quote the gamma description for the validity mask here:
 The hue of the output image is determined by the coherence level. If the user specifies an intensity image, this image is used for the image intensity, otherwise, the output image has a constant intensity except for masked regions. All pixels corresponding to pixels with a coherence or intensity  below the threshold values are set to black, i.e. (0,0,0). The intensity threshold is useful to eliminate regions of very low signal that may contain coherent interference.  If the interference is coherent, then  the correlation is high and these unwanted points would be within the unwrapping mask unless there is a threshold for minimum signal intensity.

Reply:
The coherence is turned into coherence level where the 0 level indicates the coherence values or intensity are below the threshold.

Reply:
@cirrusasf Are you suggesting specific changes? It sounds from the description you pasted that maybe the validity mask has 0 for invalid pixels, but a range of values for the rest of the pixels (rather than 1 as described in the coherence mask description). Is that the case?

Reply:
Yes. I suggest change the sentence " The coherence is then turned into a mask wherein all pixels are either 0 (don't unwrap) or 1 (unwrap)."  0 means do not calculated pixels, other than 0  means the coherence and intensity both pass the threshold values, and are included in the calculation of unwrapped-phase process. I am sure you will find the better words to express this.

Reply:
Thanks! I'll make that change.

Reply:
@cirrusasf please take a look at the edits in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/139/commits/2fa01df30071dd86c07f1a9484e6f0675a1276cd) to see if that addresses your concerns.